src/App.test.js

```diff
@@ -2,13 +2,27 @@ import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import renderer from 'react-test-renderer';
 import { Provider } from 'react-redux';
-import Store from './redux/configureStore';
+import { combineReducers, createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import missions from './redux/missions/missions';
+import rockets from './redux/rockets/rockets';
 import Missions from './components/Missions';
 import Profile from './components/Profile';
+import Rockets from './components/Rockets';
 import App from './App';
 
+const rootReducer = combineReducers({
+  missions,
+  rockets,
+});
+
+const Store = createStore(
+  rootReducer,
+  applyMiddleware(thunk),
+);
+
 describe('Header component', () => {
-  test('home page link', () => {
+  test('home page link', async () => {
```

Comment:
why are you adding an async function here? The tests still pass without them so they seem unnecessary especially since we are not doing any asynchronous operations

Reply:
I got rid of the await and I forgot to drop them as well.
Good catch!